tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,139 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+setupI = {
```

Comment:
That way we test each option individually which makes it easier to see which option causes a failure.

I'm not sure right now but I think we might do this seperately for nlte and lte configurations but that should not be your concern. Your main goal should be to design this system in  such a way that we can easily add the rest.